frontend/src/views/c-authorship.vue

```diff
@@ -893,6 +932,16 @@ export default {
       .selected-label {
         order: -1;
       }
+
+      .author-breakdown {
+        overflow-y: hidden;
+
+        &__legend {
```

Comment:
Do you think it might be clearer to add some padding/margin between the legends so it's easier to differentiate? I think the current version is fine but doing something like this might make it a bit more clear
<details>
<summary>Image</summary>

![padding](https://user-images.githubusercontent.com/45852430/224521059-e9f758c7-7131-46cb-b1b2-49ef614f3bb4.png)
</details>

Reply:
Yes, I've added some padding to differentiate the author legends more easily